CMakeLists.txt

```diff
@@ -7,12 +7,21 @@ if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
     if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 7.2)
         message(FATAL_ERROR "Your GCC version ${CMAKE_CXX_COMPILER_VERSION} is too old.")
     endif()
-elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" OR "${CMAKE_CXX_COMPILER_ID}" STREQUAL "AppleClang")
+    if (APPLE)
+        # https://software.intel.com/en-us/forums/intel-threading-building-blocks/topic/749200
+        message(WARNING "We had to drop support for GCC on OS X because it caused segfaults when used with tbb. You can continue, but don't hold us responsible for any segmentation faults.")
+        execute_process(COMMAND read -p "Press [enter] to continue anyway")
```

Comment:
Wouldn't consider this the proper way to query the user for input. In CMake you expose options (ENABLE_OSX_GCC_SUPPORT defaulting to FALSE, ...) and check for them. Don't know how IDEs would handle this (probably stall...) but its unclean anyway, so let's use options.

https://cmake.org/cmake/help/v3.9/command/option.html

Reply:
I tend to forget that "IDEs" exist. ;)

Will fix as proposed, but probably not before the holidays.

Reply:
done